board/piksiv3/rootfs-overlay/etc/init.d/S38usb_gadget

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+name="usb_gadget"
+
+VENDOR_ID="0x0000"
+PRODUCT_ID="0x0000"
```

Comment:
"Need" might be a strong word. It works fine on mac. I haven't tested anything else. I assume windows will complain.